pkg/image/context.go

```diff
@@ -52,4 +54,5 @@ type Context struct {
 	// RPMResolver responsible for resolving rpm/package dependencies
 	RPMResolver    rpmResolver
 	RPMRepoCreator rpmRepoCreator
+	Helm           registry.Helm
```

Comment:
This is definitely not ideal. We have to improve it by introducing a Registry type and / or dropping all interfaces from the context in the first place.

Reply:
Agreed. I can see a world where we go back to a Builder object which lets us wire up all of these implementation paths and keep them separate from the working data needed to build an image.